src/components/tree-detail/tree-water-needs/watering-dialog.tsx

```diff
@@ -26,7 +27,10 @@ export const WateringDialog: React.FC = () => {
 			const date = new Date(e.currentTarget.date.value);
 
 			try {
+				setIsWateringLoading(true);
 				await waterTree(amount, date);
+				setIsLastWateringsExpanded(true);
```

Comment:
should we expand the lastWaterings even if waterTree wasn't successful?

Reply:
I guess it could make sense to show that "nothing" was added